### ISSUE TITLE
Inserter category tabs: avoid unnecessary aria-label

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#68138]
### Description

This is part of the code for the category tabs in the Inserter, the ones used for patterns and media categories:

https://github.com/WordPress/gutenberg/blob/12cb6a09d2e8d8def17ab72b09a7508a11aaa154/packages/block-editor/src/components/inserter/category-tabs/index.js#L64-L73

Observe that both the visible text and the aria-label use the same value `category.label`.

Is there any reason to use an aria-label here? This isn't a good practice and any ARIA should be avoided when there's no need fo rit.
Is there any case where the visible text may be removed in any way?

Cc @WordPress/gutenberg-core 

### Step-by-step reproduction instructions

N/A

### Please confirm that you have searched existing issues in the repo.

- [x] Yes

### Please confirm that you have tested with all plugins deactivated except Gutenberg.

- [x] Yes

### Please confirm which theme type you used for testing.

- [ ] Block
- [ ] Classic
- [ ] Hybrid (e.g. classic with theme.json)
- [ ] Not sure

**Comment by Mamaduka** [comment]
Looks like an oversight, then code pattern duplication. Initially introduced in #55315

**Comment by Rishit30G** [comment]
Even I agree with the same,  content inside the tab is already accessible to screen readers and it may lead to a verbose experience.
Let me know if I should raise a PR to remove the `aria-label` 
Thanks 
cc: @afercia

**Comment by afercia** [comment]
@Rishit30G sure do feel free to raise a PR, thank you.
